### ISSUE TITLE
How can i restart learning?

### ISSUE THREAD

**Issue by asw91666** [microsoft/MeshTransformer#4]
Hello, 

Thank you for the great work.
I run this code and save model up to 10 epochs. And i want to stop learning and restart from 5 epoch. 
but 'iteration' restart from 0, not the last iteration. How can i restart learning?

https://github.com/microsoft/MeshTransformer/blob/6d5747a928b46b7e2e20d06008e489b8e52b4742/metro/tools/run_metro_bodymesh.py#L193

**Comment by kevinlin311tw** [comment]
Thanks for raising the issue. 

Our code doesn't have a way to support this operation. To resume training from a specified checkpoint, you may need to write additional code.

My high-level comments:
- You may need to save model state dict, optimizer state dict, and #epoch. 
- Make sure you correctly restore the training parameters (such as model weights, optimizer, learning rate scheduler, #epoch) to resume training.
- As we random shuffle the order of training data for every epoch, theoretically restarting from #0 iteration should not ruin the training. As long as you load the correct training parameters, I think it will be fine. 

If you make it work, please feel free to submit a PR if you would like.
